--- ../original/config/AetherCraft.cfg
+++ ../server/config/AetherCraft.cfg
@@ -56,19 +56,19 @@
     I:chargerate=128
 
     # How much of the AV an Aether Manipulator will consume (in percentage without the %, range 0-100)
-    I:consumeprecision=100
+    I:consumeprecision=30
 
     # 
     B:debug=false
 
     # 
-    B:enableflyring=true
+    B:enableflyring=false
 
     # How much AV a basic Conduit can transfer maximum.
     I:flowrate=8192
 
     # How much AV the flying ring will drain per second.
-    I:flycostpersecond=8
+    I:flycostpersecond=24
 
     # Whether stacks will get get transmuted/consumed instantly or not
     B:instantconsume=false
@@ -77,7 +77,7 @@
     B:showavalways=false
 
     # How much AV it costs for a basic Aether tool to break a block
-    D:toolcost=2.0
+    D:toolcost=8.0
 }
 
 
