--- ../original/config/magicalcrops.cfg
+++ ../server/config/magicalcrops.cfg
@@ -81,8 +81,8 @@
 ####################
 
 general {
-    I:"Change number between 1-100 for chance of getting second seed drop, 10 = 10%, 20 = 20%(default) and so on."=20
-    I:"Change to disable or enabled second seed drop, 1 = Enabled (Default), 0 = Disabled"=1
+    I:"Change number between 1-100 for chance of getting second seed drop, 10 = 10%, 20 = 20%(default) and so on."=1
+    I:"Change to disable or enabled second seed drop, 1 = Enabled (Default), 0 = Disabled"=0
     I:"Disable Crafting for Magic Fertiliser, 0 = Enabled(Default), 1 = disabled"=0
     I:"Disable ability to get Essence seeds, 0 = Enabled(Default), 1 = disabled"=0
     I:"Disable/Enable Generation of Essence ore in the Nether, 0 = Enabled(Default), 1 = disabled"=0
@@ -130,7 +130,7 @@
     I:"MOD Essence Craft Output; Tin Ore, default 6, Max 64, Min 1"=6
     I:"MOD Essence Craft Output; Uranium Ore, default 2, Max 64, Min 1"=2
     I:"MOD Essence Craft Output; Vinteum Ore, default 4, Max 64, Min 1"=4
-    I:"MineFatoryReloaded, Planter and Harvester support, set; 0 = Enabled(Default), 1 = Disable"=1
+    I:"MineFatoryReloaded, Planter and Harvester support, set; 0 = Enabled(Default), 1 = Disable"=0
     I:"Nether Essence Craft Output; Nether Brick, default 16, Max 64, Min 1"=12
     I:"Nether Essence Craft Output; Nether Warts, default 12, Max 64, Min 1"=12
     I:"Nether Essence Craft Output; Netherrack, default 16, Max 64, Min 1"=12
