--- ../original/config/magicalcrops.cfg
+++ ../server/config/magicalcrops.cfg
@@ -109,11 +109,11 @@
     I:"Essence Craft Output; Obsidian, default 8, Max 64, Min 1"=6
     I:"Essence Craft Output; Redstone dust, default 24, Max 64, Min 1"=12
     I:"MOD Essence Craft Output; Aluminum Ore, default 6, Max 64, Min 1"=6
-    I:"MOD Essence Craft Output; Ardite Ore, default 2, Max 64, Min 1"=2
+    I:"MOD Essence Craft Output; Ardite Ore, default 2, Max 64, Min 1"=1
     I:"MOD Essence Craft Output; Blue Topaz Gem, default 4, Max 64, Min 1"=4
     I:"MOD Essence Craft Output; Certus Quartz, default 4, Max 64, Min 1"=4
     I:"MOD Essence Craft Output; Chimerite, default 4, Max 64, Min 1"=4
-    I:"MOD Essence Craft Output; Cobalt Ore, default 2, Max 64, Min 1"=2
+    I:"MOD Essence Craft Output; Cobalt Ore, default 2, Max 64, Min 1"=1
     I:"MOD Essence Craft Output; Copper Ore, default 6, Max 64, Min 1"=6
     I:"MOD Essence Craft Output; Force Gem, default 8, Max 64, Min 1"=6
     I:"MOD Essence Craft Output; Lead Ore, default 4, Max 64, Min 1"=4
@@ -130,7 +130,7 @@
     I:"MOD Essence Craft Output; Tin Ore, default 6, Max 64, Min 1"=6
     I:"MOD Essence Craft Output; Uranium Ore, default 2, Max 64, Min 1"=2
     I:"MOD Essence Craft Output; Vinteum Ore, default 4, Max 64, Min 1"=4
-    I:"MineFatoryReloaded, Planter and Harvester support, set; 0 = Enabled(Default), 1 = Disable"=1
+    I:"MineFatoryReloaded, Planter and Harvester support, set; 0 = Enabled(Default), 1 = Disable"=0
     I:"Nether Essence Craft Output; Nether Brick, default 16, Max 64, Min 1"=12
     I:"Nether Essence Craft Output; Nether Warts, default 12, Max 64, Min 1"=12
     I:"Nether Essence Craft Output; Netherrack, default 16, Max 64, Min 1"=12
