--- ../original/config/AM2/AM2.cfg
+++ ../server/config/AM2/AM2.cfg
@@ -103,13 +103,13 @@
     B:ColourblindMode=false
 
     # How much the damage in Ars Magica is scaled.
-    D:DamageMultiplier=1.0
+    D:DamageMultiplier=0.8
 
     # This will toggle mana display on and off in your inventory.  Default 'O' key in game.
     B:DisplayManaInInventory=false
 
     # Generate Ars Magica towers throughout the world.  Note: Turning this off will make Ars a very lategame mod, as you will need to research all of your spells.  It will also change the recipe for spell research to require an enchanted book instead of a lost journal fragment and make the fragments drop from spirits of knowledge.
-    B:EnableWorldGen=true
+    B:EnableWorldGen=false
     I:EverstoneRepairRate=180
 
     # Set this to true to have the forge component smelt villagers into emeralds.  This counts as an attack and lowers your reputation.
@@ -121,9 +121,9 @@
     I:MMFDimensionID=-31
 
     # Set this to true to allow AM2 to get random spell names from Seventh Sanctum, and suggest them when naming spells.  Naturally, an internet connection is required.  Keep in mind, while I try to keep things family friendly, it's possible that not all names generated are so.
-    B:MoonstoneMeteorDestroyTerrain=true
+    B:MoonstoneMeteorDestroyTerrain=false
     B:NPC_Spells_Destroy_Terrain=false
-    B:Player_Spells_Destroy_Terrain=true
+    B:Player_Spells_Destroy_Terrain=false
 
     # Set this to true to enable retroactive worldgen for Ars Magica structures and ores.  *WARNING* This may break your save!  Do a backup first!  Note: This will automatically turn off after running the game once.
     B:RetroactiveWorldGen=false
