--- ../original/config/JustAnotherSpawner/WorldSettings/world/ModEntitySettings/BattleTowers.cfg
+++ ../server/config/JustAnotherSpawner/WorldSettings/world/ModEntitySettings/BattleTowers.cfg
@@ -73,80 +73,40 @@
         }
 
         ####################
-        # seasonal forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "seasonal forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # sacred springs
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "sacred springs" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # lush swamp
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "lush swamp" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # shore
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        shore {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # arctic
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        arctic {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # grove
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        grove {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # hot springs
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "hot springs" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
         # biomesoplenty
         ####################
 
         biomesoplenty {
+            ####################
+            # volcano
+            #===================
+            # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+            ####################
+
+            volcano {
+                S:"BattleTowers.Battletower Golem"=0-4-0-4
+            }
+
+            ####################
+            # boreal forest
+            #===================
+            # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+            ####################
+
+            "boreal forest" {
+                S:"BattleTowers.Battletower Golem"=0-4-0-4
+            }
+
+            ####################
+            # shrubland
+            #===================
+            # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+            ####################
+
+            shrubland {
+                S:"BattleTowers.Battletower Golem"=0-4-0-4
+            }
+
             ####################
             # swampland
             #===================
@@ -188,6 +148,16 @@
             }
 
             ####################
+            # savanna
+            #===================
+            # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+            ####################
+
+            savanna {
+                S:"BattleTowers.Battletower Golem"=0-4-0-4
+            }
+
+            ####################
             # jungle
             #===================
             # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
@@ -198,6 +168,16 @@
             }
 
             ####################
+            # woodland
+            #===================
+            # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+            ####################
+
+            woodland {
+                S:"BattleTowers.Battletower Golem"=0-4-0-4
+            }
+
+            ####################
             # junglehills
             #===================
             # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
@@ -228,6 +208,16 @@
             }
 
             ####################
+            # tundra
+            #===================
+            # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+            ####################
+
+            tundra {
+                S:"BattleTowers.Battletower Golem"=0-4-0-4
+            }
+
+            ####################
             # foresthills
             #===================
             # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
@@ -247,1275 +237,25 @@
                 S:"BattleTowers.Battletower Golem"=0-4-0-4
             }
 
-            ####################
-            # glacier
-            #===================
-            # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-            ####################
-
-            glacier {
-                S:"BattleTowers.Battletower Golem"=0-4-0-4
-            }
-
-        }
-
-        ####################
-        # highland
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        highland {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # frozenriver
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        frozenriver {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # desert oil field
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "desert oil field" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # mushroomislandshore
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        mushroomislandshore {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # snowy coniferous forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "snowy coniferous forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # storage cell
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "storage cell" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # beach
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        beach {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # icy hills
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "icy hills" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # dead forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "dead forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # mangrove
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        mangrove {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # moomoofarm
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        moomoofarm {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # brushland
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        brushland {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # coniferous forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "coniferous forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # oceanic abyss
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "oceanic abyss" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # bloody heap
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "bloody heap" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # crag
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        crag {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # deciduous forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "deciduous forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # mystic grove
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "mystic grove" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # deadlands
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        deadlands {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # shrubland
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        shrubland {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # meadow forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "meadow forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # bayou
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        bayou {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # silkglades
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        silkglades {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # corrupted sands
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "corrupted sands" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # nether
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        nether {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # tropics
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        tropics {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # river
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        river {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # heathland
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        heathland {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # spruce woods
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "spruce woods" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # plains
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        plains {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # garden
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        garden {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # maple woods
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "maple woods" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # fen
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        fen {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # seasonal spruce forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "seasonal spruce forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # junglehills
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        junglehills {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # ominous woods
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "ominous woods" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # alps mountainside
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "alps mountainside" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # boreal forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "boreal forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # wonderous woods
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "wonderous woods" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # desert
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        desert {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # orchard
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        orchard {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # canyon
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        canyon {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # quagmire
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        quagmire {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # undergarden
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        undergarden {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # frozenocean
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        frozenocean {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # alps forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "alps forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # meadow
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        meadow {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # cherry blossom grove
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "cherry blossom grove" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # pasture meadow
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "pasture meadow" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # marsh
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        marsh {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # tropics mountain
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "tropics mountain" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # origin valley
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "origin valley" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # tundra
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        tundra {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # mushroomisland
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        mushroomisland {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # wetland
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        wetland {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # chaparral
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        chaparral {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # temperate rainforest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "temperate rainforest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # savanna
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        savanna {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # thick shrubland
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "thick shrubland" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # ocean oil field
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "ocean oil field" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # snowy dead forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "snowy dead forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # dead swamp
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "dead swamp" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # ice plains
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "ice plains" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # lush desert
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "lush desert" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # rainforest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        rainforest {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # grassland
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        grassland {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # redwood forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "redwood forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # foresthills
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        foresthills {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        forest {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # timber
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        timber {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # majestic meadow
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "majestic meadow" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # shield
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        shield {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # prairie
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        prairie {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # frost forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "frost forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # coral reef
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "coral reef" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # sludgepit
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        sludgepit {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # wasteland
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        wasteland {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # witchwoodforest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        witchwoodforest {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # polar
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        polar {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # gravel beach
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "gravel beach" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # field
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        field {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # lavender fields
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "lavender fields" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # extreme hills
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "extreme hills" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # alps
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        alps {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # thicket
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        thicket {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # jungle
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        jungle {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # outback
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        outback {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # extreme hills edge
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "extreme hills edge" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # savanna plateau
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "savanna plateau" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # sky
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        sky {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # scrubland
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        scrubland {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # thinned timber
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "thinned timber" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # deserthills
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        deserthills {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # overgrown beach
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "overgrown beach" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # birch forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "birch forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # mesa
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        mesa {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # boneyard
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        boneyard {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # bog
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        bog {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # phantasmagoric inferno
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "phantasmagoric inferno" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # sublime shrubland
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "sublime shrubland" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # swampland
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        swampland {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # woodland
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        woodland {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # moor
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        moor {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # overgrown greens
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "overgrown greens" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # ocean
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        ocean {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # blessed bog
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "blessed bog" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # thinned mystic grove
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "thinned mystic grove" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # taigahills
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        taigahills {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # kelp forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "kelp forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # jade cliffs
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "jade cliffs" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # bamboo forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "bamboo forest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # thinned pasture
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "thinned pasture" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # hell
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        hell {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # forested field
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "forested field" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # oasis
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        oasis {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # badlands
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        badlands {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # steppe
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        steppe {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # autumn hills
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "autumn hills" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # volcano
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        volcano {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # dunes
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        dunes {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # tropical rainforest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "tropical rainforest" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # canyon ravine
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "canyon ravine" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # mountain
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        mountain {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # eerie
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        eerie {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # tainted land
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "tainted land" {
-            S:"BattleTowers.Battletower Golem"=0-4-0-4
-        }
-
-        ####################
-        # magical forest
-        #===================
-        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-        ####################
-
-        "magical forest" {
+        }
+
+        ####################
+        # seasonal forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "seasonal forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # sacred springs
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "sacred springs" {
             S:"BattleTowers.Battletower Golem"=0-4-0-4
         }
 
@@ -1534,56 +274,56 @@
                 S:"BattleTowers.Battletower Golem"=0-4-0-4
             }
 
-            ####################
-            # tundra
-            #===================
-            # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-            ####################
-
-            tundra {
-                S:"BattleTowers.Battletower Golem"=0-4-0-4
-            }
-
-            ####################
-            # savanna
-            #===================
-            # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-            ####################
-
-            savanna {
-                S:"BattleTowers.Battletower Golem"=0-4-0-4
-            }
-
-            ####################
-            # woodland
-            #===================
-            # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-            ####################
-
-            woodland {
-                S:"BattleTowers.Battletower Golem"=0-4-0-4
-            }
-
-            ####################
-            # shrubland
-            #===================
-            # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-            ####################
-
-            shrubland {
-                S:"BattleTowers.Battletower Golem"=0-4-0-4
-            }
-
-            ####################
-            # volcano
-            #===================
-            # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-            ####################
-
-            volcano {
-                S:"BattleTowers.Battletower Golem"=0-4-0-4
-            }
-
+        }
+
+        ####################
+        # lush swamp
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "lush swamp" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # shore
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        shore {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # arctic
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        arctic {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # grove
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        grove {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # hot springs
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "hot springs" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
         }
 
         ####################
@@ -1597,20 +337,303 @@
         }
 
         ####################
-        # net
-        ####################
-
-        net {
-            ####################
-            # boreal forest
-            #===================
-            # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
-            ####################
-
-            "boreal forest" {
-                S:"BattleTowers.Battletower Golem"=0-4-0-4
-            }
-
+        # highland
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        highland {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # frozenriver
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        frozenriver {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # mushroomislandshore
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        mushroomislandshore {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # desert oil field
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "desert oil field" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # snowy coniferous forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "snowy coniferous forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # storage cell
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "storage cell" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # icy hills
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "icy hills" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # beach
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        beach {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # dead forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "dead forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # mangrove
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        mangrove {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # eerie
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        eerie {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # moomoofarm
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        moomoofarm {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # brushland
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        brushland {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # coniferous forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "coniferous forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # oceanic abyss
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "oceanic abyss" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # crag
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        crag {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # bloody heap
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "bloody heap" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # deciduous forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "deciduous forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # mystic grove
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "mystic grove" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # deadlands
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        deadlands {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # shrubland
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        shrubland {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # meadow forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "meadow forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # bayou
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        bayou {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # silkglades
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        silkglades {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # corrupted sands
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "corrupted sands" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # nether
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        nether {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # tropics
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        tropics {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # river
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        river {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # heathland
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        heathland {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # spruce woods
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "spruce woods" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
         }
 
         ####################
@@ -1624,6 +647,86 @@
         }
 
         ####################
+        # plains
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        plains {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # garden
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        garden {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # maple woods
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "maple woods" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # fen
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        fen {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # seasonal spruce forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "seasonal spruce forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # junglehills
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        junglehills {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # ominous woods
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "ominous woods" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # alps mountainside
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "alps mountainside" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
         # snowy beach
         #===================
         # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
@@ -1633,6 +736,896 @@
             S:"BattleTowers.Battletower Golem"=0-4-0-4
         }
 
+        ####################
+        # boreal forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "boreal forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # wonderous woods
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "wonderous woods" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # desert
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        desert {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # orchard
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        orchard {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # canyon
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        canyon {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # quagmire
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        quagmire {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # undergarden
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        undergarden {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # meadow
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        meadow {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # frozenocean
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        frozenocean {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # alps forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "alps forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # cherry blossom grove
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "cherry blossom grove" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # pasture meadow
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "pasture meadow" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # marsh
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        marsh {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # tropics mountain
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "tropics mountain" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # origin valley
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "origin valley" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # mushroomisland
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        mushroomisland {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # tundra
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        tundra {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # wetland
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        wetland {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # chaparral
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        chaparral {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # savanna
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        savanna {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # temperate rainforest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "temperate rainforest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # thick shrubland
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "thick shrubland" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # ocean oil field
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "ocean oil field" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # snowy dead forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "snowy dead forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # dead swamp
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "dead swamp" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # ice plains
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "ice plains" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # lush desert
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "lush desert" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # rainforest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        rainforest {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # grassland
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        grassland {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # redwood forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "redwood forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # foresthills
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        foresthills {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        forest {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # majestic meadow
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "majestic meadow" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # timber
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        timber {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # shield
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        shield {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # prairie
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        prairie {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # frost forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "frost forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # coral reef
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "coral reef" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # wasteland
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        wasteland {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # sludgepit
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        sludgepit {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # witchwoodforest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        witchwoodforest {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # polar
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        polar {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # field
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        field {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # gravel beach
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "gravel beach" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # extreme hills
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "extreme hills" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # lavender fields
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "lavender fields" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # alps
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        alps {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # thicket
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        thicket {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # jungle
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        jungle {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # outback
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        outback {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # savanna plateau
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "savanna plateau" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # extreme hills edge
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "extreme hills edge" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # sky
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        sky {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # scrubland
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        scrubland {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # thinned timber
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "thinned timber" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # deserthills
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        deserthills {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # overgrown beach
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "overgrown beach" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # birch forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "birch forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # boneyard
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        boneyard {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # mesa
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        mesa {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # bog
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        bog {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # sublime shrubland
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "sublime shrubland" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # swampland
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        swampland {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # phantasmagoric inferno
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "phantasmagoric inferno" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # woodland
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        woodland {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # glacier
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        glacier {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # moor
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        moor {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # overgrown greens
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "overgrown greens" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # ocean
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        ocean {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # blessed bog
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "blessed bog" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # thinned mystic grove
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "thinned mystic grove" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # taigahills
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        taigahills {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # kelp forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "kelp forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # jade cliffs
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "jade cliffs" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # thinned pasture
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "thinned pasture" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # bamboo forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "bamboo forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # hell
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        hell {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # forested field
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "forested field" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # oasis
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        oasis {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # badlands
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        badlands {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # tainted land
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "tainted land" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # steppe
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        steppe {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # autumn hills
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "autumn hills" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # magical forest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "magical forest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # volcano
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        volcano {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # dunes
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        dunes {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # tropical rainforest
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "tropical rainforest" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # canyon ravine
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        "canyon ravine" {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
+        ####################
+        # mountain
+        #===================
+        # Editable Format: SpawnWeight-SpawnPackSize-MinChunkPackSize-MaxChunkPackSize
+        ####################
+
+        mountain {
+            S:"BattleTowers.Battletower Golem"=0-4-0-4
+        }
+
     }
 
 }
